roles/engineblock/tasks/test.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Create install directory
+  file: "path={{ engineblock_selenium_install_dir }}/selenium state=directory recurse=yes"
+
```

Comment:
A minor cosmetic nitpick: Could you use the same indentation style as the other tasks?

Reply:
Thanks, changed the style